src/guest/start.c

```diff
@@ -1142,6 +1146,11 @@ int start_guest(char *name)
 	if (val != NULL && (strcmp("", val) != 0))
 		run_thermal_mediation_daemon(val);
 
+	val = g_key_file_get_string(gkf, g->name, g->key[CAMERA_MED], NULL);
+	if (val != NULL && (strcmp("", val) != 0)) {
+		run_camera_mediation_daemon("./scripts/stream");
```

Comment:
do not assume stream is under where vm-manager is executed. use run_camera_mediation_daemon(val); That is why we need CAMERA_MED sub key